### PR TITLE
sabakan-cryptsetup: retry sabakan API calls.

### PR DESCRIPTION
**Pull request by ymmt2005**

### CODE REVIEW

File: pkg/sabakan-cryptsetup/cmd/driver.go [hunk 1]
```diff
@@ -152,7 +156,18 @@ func (d *Driver) setupDisk(ctx context.Context, disk Disk, tpmKek []byte) error
...
+	log.Error("failed to retrieve key from sabakan", map[string]interface{}{
+		log.FnError: err,
+		"disk":      disk.Name(),
+		"try":       retries + 1,
+	})
+	if retries == maxRetry {
+		return err
+	}
+	retries++
+	time.Sleep(time.Duration(retries) * time.Second * 2)
```
Comment (tapih) [inline pkg/sabakan-cryptsetup/cmd/driver.go]: Just to confirm, do you intend to make sleep time longer if retry happens multiple times?

Comment (ymmt2005) [reply pkg/sabakan-cryptsetup/cmd/driver.go]: Yes.

File: pkg/sabakan-cryptsetup/cmd/driver.go [hunk 2]
```diff
@@ -152,7 +156,18 @@ func (d *Driver) setupDisk(ctx context.Context, disk Disk, tpmKek []byte) error
...
+
+	log.Error("failed to retrieve key from sabakan", map[string]interface{}{
+		log.FnError: err,
+		"disk":      disk.Name(),
+		"try":       retries + 1,
+	})
+	if retries == maxRetry {
+		return err
+	}
+	retries++
```
Comment (tapih) [inline pkg/sabakan-cryptsetup/cmd/driver.go]: `retries++` should be before `if retries == maxRetry` block or else retry happens (maxRetry + 1) times.

File: pkg/sabakan-cryptsetup/cmd/driver.go [hunk 3]
```diff
@@ -183,5 +198,21 @@ func (d *Driver) formatDisk(ctx context.Context, disk Disk, f *os.File, tpmKek [
...
+	}
+	log.Error("failed to send key to sabakan", map[string]interface{}{
+		log.FnError: err,
+		"disk":      disk.Name(),
+		"try":       retries + 1,
+	})
+	if retries == maxRetry {
+		return err
+	}
+	retries++
```
Comment (tapih) [inline pkg/sabakan-cryptsetup/cmd/driver.go]: same here.

File: pkg/sabakan-cryptsetup/cmd/driver.go (hunk 2 above)
Comment (ymmt2005) [reply pkg/sabakan-cryptsetup/cmd/driver.go]: No.  Retries can happen exactly at most `maxRetry` times.
Note that the first try is *not* a retry.

**Pull request merged**